lib/v2/gcores/collection.js

```diff
@@ -0,0 +1,119 @@
+/* refer to: ./tag.js (author: @StevenREC0) */
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    // get params
+    const { collection } = ctx.params;
+    // make a request
+    const baseUrl = `https://www.gcores.com/gapi/v1/collections/${collection}`;
+    const url = `${baseUrl}/articles?sort=-published-at`;
+    const baseRes = await got(baseUrl);
+    const res = await got(url);
+    // resolve data
+    const feedTitle = baseRes.data.data.attributes.title;
+    const list = res.data.data.map((item) => ({
+        url: `articles/${item.id}`,
+        title: item.attributes.title,
+    }));
+
+    if (list.length > 0 && list.every((item) => item.url === undefined)) {
+        throw new Error('Article URL not found! Please submit an issue on GitHub.');
+    }
+
+    const out = await Promise.all(
+        list.map((item) => {
+            const articleUrl = `https://www.gcores.com/${item.url}`;
+            return ctx.cache.tryGet(articleUrl, async () => {
+                const itemRes = await got({
+                    method: 'get',
+                    url: articleUrl,
+                });
+
+                const itemPage = itemRes.data;
+                const $ = cheerio.load(itemPage);
+                let articleData = await got(`https://www.gcores.com/gapi/v1/${item.url}?include=media`);
```

Comment:
The JSON response of `res` in line 12 already contains the full articles (blocks) and their media (entityMap) in `data.attributes.content`. 

You don't need to visit `https://www.gcores.com/${item.url}` and `https://www.gcores.com/gapi/v1/${item.url}?include=media` thereafter.

Reply:
That means I need to format all content blocks into html by myself? I will take a look at `data.attributes.content`. Thanks